File: src/test/java/com/yah/StudentTest.java
package com.yah;

import com.yah.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestScoreTest(){
       Student student = new Student("Bob", 50, 84);
        Assertions.assertEquals(84 , student.highest());
    }

    @Test
    public void averageTest(){
        Student student = new Student("Bob", 50, 84);
        Assertions.assertEquals((50+84)/2 , student.getAverage());
    }
}

File: src/test/kotlin/com/yah/StudentTest.kt
package com.yah.kotlin

import com.yah.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest(){
        val student = Student("Mi", 84, 63)
        Assertions.assertEquals(84 , student.highest())
    }

    @Test
    fun averageTest(){
        val student = Student("Mi", 84, 63)
        Assertions.assertEquals((84+63)/2 , student.getAverage())
    }

    @Test
    fun gradingTest(){
        val student = Student("Mi", 84, 63)
        Assertions.assertEquals('C' , student.grading())
    }
}